src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -76,7 +76,7 @@ export class IndexPattern implements IIndexPattern {
   private shortDotsEnable: boolean = false;
   private fieldFormats: FieldFormatsStartCommon;
   private fieldAttrs: FieldAttrs;
-  private runtimeFieldMap: Record<string, RuntimeField>;
+  public runtimeFieldMap: Record<string, RuntimeField>;
```

Comment:
@mattkime could you please help me a little with that. Is it ok, or we should get runtimeFieldMap using the different way?